### PR TITLE
Issue/#3 - create UI for analog clock

### PR DESCRIPTION
**Pull request by gnrlbzik**
GH Issue: https://github.com/KorolevskiTigr2/BYBYBY/issues/3

### CODE REVIEW

**Comment by gnrlbzik** [comment]
nit: we still need to fix folder and file structure

- `Project_root/Components/Clock dial/Clock_dial.css` should be as following`components/clock-dial/clock-dial.css`
- `Project_root/Components/Clock dial/Clock_dial.html` should be as following`components/clock-dial/clock-dial.html`

File: Project_root/Components/Clock dial/Clock_dial.css [hunk 1]
```diff
@@ -0,0 +1,168 @@
+* {
```
[the remark this answers is not in the document]
Comment (gnrlbzik) [reply Project_root/Components/Clock dial/Clock_dial.css:1]: this should be fixed.

**Comment by gnrlbzik** [comment]
> nit: we still need to fix folder and file structure
> 
> * `Project_root/Components/Clock dial/Clock_dial.css` should be as following`components/clock-dial/clock-dial.css`
> * `Project_root/Components/Clock dial/Clock_dial.html` should be as following`components/clock-dial/clock-dial.html`

Just a reminder, fix above naming please @KorolevskiTigr2

**Comment by KorolevskiTigr2** [comment]
> > nit: we still need to fix folder and file structure
> > 
> > * `Project_root/Components/Clock dial/Clock_dial.css` should be as following`components/clock-dial/clock-dial.css`
> > * `Project_root/Components/Clock dial/Clock_dial.html` should be as following`components/clock-dial/clock-dial.html`
> 
> ‎Просто напоминание, исправьте выше именования, пожалуйста‎ @KorolevskiTigr2

gotovo :)

**Comment by gnrlbzik** [comment]
> > > nit: we still need to fix folder and file structure
> > > 
> > > * `Project_root/Components/Clock dial/Clock_dial.css` should be as following`components/clock-dial/clock-dial.css`
> > > * `Project_root/Components/Clock dial/Clock_dial.html` should be as following`components/clock-dial/clock-dial.html`
> > 
> > 
> > ‎Просто напоминание, исправьте выше именования, пожалуйста‎ @KorolevskiTigr2
> 
> gotovo :)

Not exactly, please double check the request, it seems you have few more things to change,!

Please refer to the my example.

**Comment by gnrlbzik** [comment]
> > 
> 
> Снимок экрана (10) I don't know, why this changes does not work

what change are you referring to?

**Pull request merged**